django/contrib/admin/static/admin/css/forms.css

```diff
@@ -303,8 +303,10 @@ body.popup .submit-row {
     padding: 10px 15px;
     height: 15px;
     line-height: 15px;
-    margin: 0 0 0 5px;
+    margin: 0 5px 5px 5px;
```

Comment:
Left margin is unnecessary after b4817d20b9e55df30be0b1b2ca8c8bb6d61aab07. Please remove it
```suggestion
    margin: 0 5px 5px 0px;
```
so
![image](https://user-images.githubusercontent.com/2865885/179917325-8955d670-0f32-4c0a-b756-0d7469620bfd.png)

instead of
![image](https://user-images.githubusercontent.com/2865885/179917160-358e3ec5-3f6d-4eaa-a38e-65a4a27eb1cf.png)

Reply:
We should remove the bottom margin, but the bottom gap is the case for the rest of the buttons also. It is related to the case where a button is alone only?

<img src="https://user-images.githubusercontent.com/39882869/180059064-eb8819a7-59c1-4858-aa39-2c0683030c96.png" width="400" />

If the delete button is present, it works well and the gaps are the same:

<img src="https://user-images.githubusercontent.com/39882869/180061206-22a57c0e-6058-424e-9338-dc77e7c8c0a2.png" width="400" />

As for me, it is OK, but maybe someone thinks another way. :)

Reply:
I'm confused, your comment is probably a reply to another request. I was asking to remove the left margin from the `Close` button not to remove the bottom margin from the `Delete` button :face_with_head_bandage: